app/assets/stylesheets/pages/_home.scss

```diff
@@ -1,9 +1,10 @@
 // Specific CSS for your home-page
-.home-page {
-  body {
-    font-family: 'Work Sans',sans-serif;
-    background-image: url("https: //images.immediate.co.uk/production/volatile/sites/4/2021/08/mountains-7ddde89.jpg?quality=90&resize=768,574");
-  }
```

Comment:
Can we keep this one nested within the .home-page class? It will stop weird stuff happening on other pages.

Reply:
about to push change!